resolver/type_syntax.cc

```diff
@@ -784,6 +785,7 @@ TypeSyntax::ResultType TypeSyntax::getResultTypeAndBind(core::MutableContext ctx
                     e.addErrorSection(
                         core::ErrorSection(core::ErrorColors::format("`{}` will not work in the runtime type system.",
                                                                      recvi->symbol.data(ctx)->show(ctx) + "[...]")));
+                    e.replaceWith("Use the correct syntax", recvi->loc, corrected.data(ctx)->show(ctx));
```

Comment:
Instead of saying "the correct syntax" can we give someone an indication of what edit will be made? The context is that the first arg to replaceWith is the entire message that will be shown to the user in the code action drop down. it's not a great experience to have to apply the edit to figure out what the edit is going to do.